backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -408,6 +408,10 @@ public static bool IsProjectInDraft(this Entity.Project project, ProjectOptions
         /// <returns></returns>
         public static bool IsProjectClosed(this Entity.Project project)
         {
+            if(project?.Status == null)
```

Comment:
This was causing failures for me when submitting parcels - it shouldn't.

Reply:
This would be the result of it not being loaded correctly by the caller.  It doesn't really make sense to return false if the `project == null`.  It should throw an `ArgumentException` in that case.

You can also simplify to `return project.Status?.IsTerminal ?? false`.  Although I think it probably makes more sense to throw an exception as the `Status` hasn't been loaded correctly for this function.

Reply:
I think I'm missing something here, why is this even being called during parcel submission? this is on a vanilla DB with no projects. If I throw an exception here the submission will fail. However, I get that returning false here may not make sense. I'll need to dig deeper on this I guess.

Reply:
nevermind, just repro'd this again. This occurs when trying to update a parcel that is in a project. Digging deeper.

Reply:
@Fosol this was introduced by this change:

```
// update only an active project with any financial value changes.
if (!String.IsNullOrWhiteSpace(parcel.ProjectNumber) && (!this.Context.Projects.FirstOrDefault(p => p.ProjectNumber == parcel.ProjectNumber)?.IsProjectClosed() ?? false))
{
    this.Context.UpdateProjectFinancials(parcel.ProjectNumber);
}
```

correct me if I'm wrong, but won't this always fail since this.Context.Projects is not eager loading the Project.Status?

Reply:
Well, going ahead with adding the include here for now since it allows me to remove the above if statement.

Reply:
We could load the Project if it is required at this point